samtranslator/schema/schema.py

```diff
@@ -176,65 +223,61 @@ class FunctionSqsEvent(BaseModel):
     Properties: FunctionSqsProperties
 
 
-class FunctionApiProperties(BaseModel):
-    Auth: Unknown
-    Method: Unknown
-    Path: Unknown
-    RequestModel: Unknown
-    RequestParameters: Unknown
-    RestApiId: Unknown
+class FunctionResourcePolicy(BaseModel):
+    AwsAccountBlacklist: Optional[List[ResourcePolicy]]
```

Comment:
Are you sure this is correct for all? It passed with `Optional[List[str]]` for the state machine.

Reply:
<img width="797" alt="Screen Shot 2022-11-21 at 2 59 32 PM" src="https://user-images.githubusercontent.com/28763956/203174937-c73dafcf-4bb8-4bfc-9c8a-db0347f4eecf.png">

Reply:
That's for `CustomStatements`, which by it's name I assume stands for policy statement, so a dict makes sense. Is that the case for all others?

Reply:
I'll change the statemachine one to match function. I checked document for SFN, it's also a dict or string.